.github/workflows/tests-prophet.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
       max-parallel: 4
       fail-fast: false
       matrix:
-        python-version: ['3.8']
+        python-version: ['3.12']
```

Comment:
prophet itself only appears to support until 3.11

https://github.com/facebook/prophet/blob/main/python/pyproject.toml